scripts/wasmd/template/.wasmcli/cosmos1fa7hj49pf8uzc4m0lw5swjhhl5th2484dc82dl.address

```diff
@@ -0,0 +1 @@
+eyJhbGciOiJQQkVTMi1IUzI1NitBMTI4S1ciLCJjcmVhdGVkIjoiMjAyMC0wNy0yMyAxNTowNTo0OC4yMDIyMjEgKzAwMDAgVVRDIG09KzAuNDA3MTQ2NDAxIiwiZW5jIjoiQTI1NkdDTSIsInAyYyI6ODE5MiwicDJzIjoiZXhEVEM1dGVraTVyMDg4ZyJ9.AluhnvIPJAJZZIUwCATLZ2eQNBy-bqUMOwa1QVokRPU4MWOsuxx-TQ.3vXpP5cMQEXdSBko.l5BzPkI5_DUm-iUBM9T9po-9YXYmGK62eieeFVA_WzESZPYxk51fQq_8DCdhh6Y6DhmmvQXPv3hjqQgIgdxPRBFlJwNawrqLElWIVuiS02tAjwQXAw6to0sdNtJc5r8Fzr82aD_lRMDiJ5ZoJ6i3wHLgmfwtOH0JdGwGFwiEQYuWX0SK0Ms2LHUydmsriX6MbHik24R061Qc3dR0GfclCb9dP_jRvXBhen8EJyt3cM3opYZ5lqDcLv_OQnDaFb4.Ctl177TJVkOiSYSaG3IBBg
```

Comment:
I think there is a typo here :stuck_out_tongue: 

just kidding... had to saw something when scrolling over tons of auto-gen code